README.md

```diff
@@ -1,7 +1,7 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_get_data_DOM/)
+    - [DEMO LINK](https://gboreyko.github.io/js_get_data_DOM/)
```

Comment:
Add the demo link to the PR description 
<img width="977" alt="image" src="https://github.com/user-attachments/assets/13566d76-6ec2-43da-93c8-b41bdb11c0c3">